src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -114,9 +114,10 @@ end
 """
     NetCDFOutputWriter{D, O, I, T, S} <: AbstractOutputWriter
 
-An output writer for writing to NetCDF files.
+An output writer for writing model data to NetCDF files.
 """
-mutable struct NetCDFOutputWriter{D, O, T, S, A} <: AbstractOutputWriter
+mutable struct NetCDFOutputWriter{D, O, T, S, A, M} <: AbstractOutputWriter
```

Comment:
In the other functions you tended to put M at the beginning. Now you have it at the end.  I am fine with either but is there a reason for the switch?

Reply:
Just paranoia that the type parameter `D` might've been used somewhere. Adding a new type parameter at the end is conservative. In the case of the `JLD2OutputWriter`, I knew that the type parameter was not used.